.cm/gitstream.cm

```diff
@@ -72,4 +72,8 @@ changes:
   # Sum all the line removed in the PR
   deletions: {{ branch.diff.files_metadata | map(attr='deletions') | sum }}
   # Calculate the ratio of new code
-  ratio: {{ (changes.additions / (changes.additions + changes.deletions)) * 100 | round(2) }}
\ No newline at end of file
+  ratio: {{ (changes.additions / (changes.additions + changes.deletions)) * 100 | round(2) }}
+
+has:
+  screenshot_link: {{ pr.description | includes(regex=r/!\[.*\]\(.*(jpg|svg|png|gif|psd).*\)/) }}
```

Comment:
what's the purpose of these changes?

Reply:
This is for gitStream, a custom function in the config I missed, resulting in gitStream bot commenting screenshot request despite already included one in PR description.